scripts/enrich_rulings_ai.py

```
@@ -183,12 +181,12 @@ def convert_external_ruling_to_standard_format(
 def enrich_rulings(rulings_data: list[dict[str, any]]) -> list[dict[str, any]]:
     """
     Enriches a list of ruling dictionaries with AI-generated metadata such as related card codes, provenance details, question-and-answer extraction, and tags.
-    
+
     Each ruling is processed to ensure required fields are present, selects the most informative text for AI analysis, and applies AI placeholder functions to update related cards, provenance, Q&A structure, and tags. Rulings lacking suitable text for enrichment are skipped but included in the output.
```

Ruff (0.11.9)

Line too long (302 > 100)

(E501)